kustomization/overlays/develop/develop-patch.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: chat-app-service
+  namespace: my-app-namespace
+spec:

```

Comment:
No separate namespace for the environments, no separate resources?

Reply:
Modified now, added separate namespaces for diff environments.

Reply:
There are still hard coded namespaces in manifests and I could not find any resources for two different environments.